README.md

```diff
@@ -61,8 +61,8 @@ You can configure auth parameters passing them like this:
 
 ```js
 const configuration = OneSignal.createConfiguration({
-    userAuthKey: '<YOUR_USER_AUTH_KEY>',
-    restApiKey: '<YOUR_REST_API_KEY>',
+    userKey: '<YOUR_USER_KEY_TOKEN>',
+    appKey: '<YOUR_APP_KEY_TOKEN>',
```

Comment:
Just double-checking that this change is ok. I recall the SDK interface follows the old (e.g. `userAuthKey` and `restApiKey`) key names, so I'm not sure why this would change now.

Reply:
I manually updated those REAME files in https://github.com/OneSignal/onesignal-node-api/pull/81. The README is out of date, and we should down the line modify the files in the generator repo.

Reply:
Maybe you can pull that commit.